docs/index.md

```diff
@@ -37,7 +37,7 @@ terraform {
 
   required_providers {
     ably = {
-      source = "ably/ably"
+      source = "hashicorp/ably"
```

Comment:
Huh any one know what happened here? I think running tfplugindocs made this change. Did the provider move to under hashicorp?

Reply:
I think this one should remain ably/ably.

Reply:
Our examples use `hashicorp/ably` too so this may be intentional? @kavalerov

Reply:
It should definitely be ably/ably as we don't host our provider in the hashicorp namespace on the terraform registry. 
We should update the examples as well in this case as they're incorrect. 
Probably a relic from when we created the examples before working on provider implementation.

Reply:
534d7565 Shows what may be a manual update to the file. Need to see how to set it to ably and stick

Reply:
So it seems to read the examples. Changing the examples fixed the issue.